lib/standard/string.nit

```diff
@@ -2207,6 +2216,12 @@ extern class NativeString `{ char* `}
 	# Creates a new NativeString with a capacity of `length`
 	new(length: Int) is intern
 
+	# Returns a char* starting at `index`.
+	#
+	# WARNING: Unsafe for extern code, use only for temporary
```

Comment:
The other one was a NOTE:, this one is a WARNING: does it mean that there is a different level of warning here?